### ISSUE TITLE
pip installation not working

### ISSUE THREAD

**Issue by kain88-de** [MDAnalysis/pmda#42]
### Expected behaviour

installation works

### Actual behaviour

ImportError: No module named 'pmda'

### Code to reproduce the behaviour

```bash
conda create -n test --yes
source activate test
pip install pmda
python -c 'import pmda'
```

**Comment by orbeckst** [comment]
I have a fix in PR #43. If you're ok with it I'll quickly ship a 0.1.1 today.

**Comment by kain88-de** [comment]
Thanks. Please go ahead with the release.

**Comment by orbeckst** [comment]
I published 0.1.1 and it is visible 

However,

**Comment by orbeckst** [comment]
Tried it again and it works now. Maybe pypi just needed an hour... all good.
